ritmap/ritmap.github.io
# Add GIS fuel cell and garden, and miller road

**Changes**
* Add Miller Road
* Add GIS Fuel cell
* Add GIS Rooftop Garden

**API Changes**
This change does not introduce API changes of any kind

**Testing**
* Ensure the syntax of data.js is correct
* Ensure the additional locations appear on the map with their associated images, reasonably close to where they are expected to be.
